logger.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import logging.handlers
+import os
+from datetime import datetime
+
+def get_logfile_path():
+    if   os.name == "posix": log_path = "log/"
+    elif os.name == "nt"   : log_path = ".\\log\\"
+    log_file = "test.txt"
+    filepath = log_path + log_file
+    if not os.path.exists(log_path):
+        os.makedirs(log_path)
+    return filepath
```

Comment:
`log/` is used here. Is `./log/` unsafe on POSIX platforms?

Maybe `text.txt` is not a quite suitable name for a log file...

Pathlib requires Python ≥ 3.5. I think it's OK because f-strings have been already used elsewhere, which require Python ≥ 3.6.

```suggestion
from pathlib import Path

def get_logfile_path(log_file="test.txt"):
    file_path = os.path.join(".", "log", log_file)
    # https://stackoverflow.com/questions/273192/how-can-i-safely-create-a-nested-directory
    Path(file_path).mkdir(parents=True, exist_ok=True)
    return file_path
```

Reply:
Thans for yoursuggestion! I'll take it.

Reply:
I've changed the code to the following:
https://github.com/pusaitou/mikochiku_alarm/pull/57/commits/5efc84ebb06f91a667be6fbe0c43801f78c8da4d#diff-3000fb145a56b8b3c44b6da2269d8c23R9-R12

+ fetch logfile path from setting.json to share with other modules.
+ Assuming that the log file name is fixed, I changed it so that the caller cannot change the file name.
+ When `Path("./log/log.txt").mkdir` is executed, a directory (not a file) named 'log.txt'  is created. So changed to `Path.mkdirs("./log")`.